plugins/inputs/influxdb_v2_listener/influxdb_v2_listener.go

```diff
@@ -294,6 +297,15 @@ func (h *InfluxDBV2Listener) handleDefault() http.HandlerFunc {
 	}
 }
 
+func (h *InfluxDBV2Listener) handlePing() http.HandlerFunc {
+	return func(res http.ResponseWriter, _ *http.Request) {
+		defer h.pingServed.Incr(1)
+		res.Header().Set("X-Influxdb-Build", "Telegraf")
```

Comment:
```suggestion
		res.Header().Set("X-Influxdb-Build", "telegraf")
```

Not sure if "telegraf" makes sense, the docs say it is "The type of InfluxDB build.".